calculator-funky-version.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react'
```

Comment:
It's fine to hang onto things while developing but you should never commit test/debug code to the repo. This could end up in master and on a production server which would be at best confusing, at worst cause something to crash.